subprojects/jacoco/src/testFixtures/groovy/org/gradle/testing/jacoco/plugins/fixtures/JacocoCoverage.groovy

```diff
@@ -46,6 +53,7 @@ final class JacocoCoverage {
         final static CHECK_INTRODUCED = new JacocoVersion(0, 6, 3)
         final static SUPPORTS_JDK_8 = new JacocoVersion(0, 7, 0)
         final static SUPPORTS_JDK_9 = new JacocoVersion(0, 7, 8)
+        final static SUPPORTS_JDK_14 = new JacocoVersion(0, 8, 5)
```

Comment:
Based on SUPPORTS_JDK_8 above, it looks like we can drop all versions below 0.7.0 since we're not testing them anyways.

Reply:
True, some of them are no longer being tested against. A separate PR: https://github.com/gradle/gradle/pull/12397